inc/admin/options.php

```diff
@@ -885,8 +883,8 @@ function rocket_settings_callback( $inputs ) {
 		$inputs['cloudflare_domain'] = '';
 	}
 
-	$inputs['cloudflare_devmode'] = ( is_numeric( $inputs['cloudflare_devmode'] ) ) ? (int) $inputs['cloudflare_devmode'] : 0;
-	$inputs['cloudflare_auto_settings'] = ( is_numeric( $inputs['cloudflare_auto_settings'] ) ) ? (int) $inputs['cloudflare_auto_settings'] : 0;
+	$inputs['cloudflare_devmode'] = ( isset( $inputs['cloudflare_devmode'] ) && is_numeric( $inputs['cloudflare_devmode'] ) ) ? (int) $inputs['cloudflare_devmode'] : 0;
+	$inputs['cloudflare_auto_settings'] = ( isset( $inputs['cloudflare_auto_settings'] ) && is_numeric( $inputs['cloudflare_auto_settings'] ) ) ? (int) $inputs['cloudflare_auto_settings'] : 0;
```

Comment:
Yes, when my CF credentials when I saved CF settings with a wrong API key and wrong email, I got a warning without those `isset()`s.